Kudu.Core/Infrastructure/LockFile.cs

```diff
@@ -287,27 +285,6 @@ private void OnLockReleasedInternal(object sender, FileSystemEventArgs e)
             }
         }
 
-        private static bool IsFileSystemReadOnly()
```

Comment:
we have a test file for FileSystemHelpers.

i was thinking other test case already cover this code path, so do not want to duplicate the effort. but sure that i can add one test case to just test this function.